src/boxplot.jl

```diff
@@ -21,6 +21,25 @@ notch_width(q2, q4, N) = 1.58 * (q4-q2)/sqrt(N)
     bw == nothing && (bw = 0.8)
     @assert whisker_width == :match || whisker_width >= 0 "whisker_width must be :match or a positive number"
     ww = whisker_width == :match ? bw : whisker_width
+
+    nr, nc = size(y) # size(y) == (6,)
```

Comment:
I copied this code from box.jl, but it's giving me an error at the moment because the return is a 1-element tuple.

I tried adding in the `grouped_xy()` function from `bar.jl` as well, but that doesn't seem to work. I'm guessing the `x` and `y` pulled in by `g.args...` in:

```
@recipe function f(g::GroupedBar)
    x, y = grouped_xy(g.args...)
     ...
```

Is different than from `@recipe function f(::Type{Val{:boxplot}}, x, y, z;...)`, but I don't understand enough to know how to fix.